package.json

```diff
@@ -78,7 +78,7 @@
     "compression-webpack-plugin": "^0.3.0",
     "copy-webpack-plugin": "^1.1.1",
     "css-loader": "^0.23.1",
-    "cucumber": "^0.10.1",
+    "cucumber": "^0.9.1",
```

Comment:
Npm printed a message, that "protractor-cucumber-framework" in version "^0.4.0" requires cucumber in version "0.9.1".